treeherder/log_parser/failureline.py

```diff
@@ -94,13 +95,20 @@ def get_kwargs(failure_line):
             if key in failure_line}
 
 
+def create_failure_line(job_log, failure_line):
+    fl = FailureLine.objects.create(repository=job_log.job.repository,
+                                    job_guid=job_log.job.guid,
+                                    job_log=job_log,
+                                    **get_kwargs(failure_line))
+    if "group" in failure_line:
+        group, saved = Group.objects.get_or_create(name=failure_line["group"])
+        group.failure_lines.add(fl)
```

Comment:
I was surprised that it did, yeah.  You can see from the tests that it does it just fine.  I had originally put a save in there, but tried it without and lo-and-behold it worked fine.